packages/gestalt-design-tokens/tokens/font/base.json

```diff
@@ -27,10 +27,6 @@
       },
       "semibold": {
```

Comment:
Can we just call this "bold" since we no longer have another option? It keeps it backward compatible, esp with the tokens. So now the bold token just points to 600 instead of 700

Reply:
@ayeshakmaz That was my first intention.

The Typography  page mentions semibold so I thought we wanted to keep that difference. Still think I should move it to bold?
 


https://gestalt.pinterest.systems/typography#Font-weight
![Screen Shot 2022-06-28 at 7 47 32 PM](https://user-images.githubusercontent.com/10593890/176322695-c954dc71-7822-4c53-824c-a3e929ad5781.png)

Reply:
Oh I think in iOS the phrasing they use is semibold, same way in android they use "medium"

Reply:
So it's not because of us, but the platform

Reply:
[MDN defines "bold" as 700](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#values). Not sure if we want to break with that convention? (I could very easily be talked into it though)

Reply:
Semibold is the appropriate term to use here. We're moving away from bold and to semibold.

Reply:
ahh ok, we need to label this as a breaking change then because we're removing a design token

Reply:
Will the prop on Text also be changing?

Reply:
It may be worth chatting with Hector before we remove the bold weight token. I think there's consideration of exploring Text using bold down the road.

Reply:
@ponori @ayeshakmaz I'll keep the token, we can deprecate down the road if there's more discussion needed around that